pydoctor/test/test_astbuilder.py

```diff
@@ -282,7 +290,9 @@ class D(C):
     addsrc(system)
     D = system.allobjects['d.D']
     assert isinstance(D, model.Class)
-    assert D.bases == ['a.A']
+    # An older version of this test expected a.A as the result.
+    # Read the comment in test_aliasing() to learn why this was changed.
+    assert D.bases == ['c.C']
```

Comment:
To make it easier to understand what we have here, maybe use `_private_stuff` instead of just `a`.

Also, instead of `c` use `public_stuff`.

And instead of `b` use "_private_agregator` or something like that.

I hope that in this way it will make it easier to see that we don't want to expose a, since a is a private module